onboarding/src/Components/Dashboard/DashboardPage.js

```diff
@@ -0,0 +1,117 @@
+import React, { useRef, useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { getPath } from "../utils.js";
+import { v4 as uuidv4 } from "uuid";
+
+import Navbar from "../Navbar";
+import LeftMenu from "../LeftMenu";
+import PageAddressBar from "../PageAddressBar";
+import Employee from "./Employee";
+
+//import "../static/looper/stylesheets/theme.min.css";
+//import "../static/looper/stylesheets/theme-dark.min.css";
+
+function DashboardPage(props) {
+  const packageIdRef = useRef(0);
+
+  const [isEmployee, setIsEmployee] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [employees, setEmployees] = useState([]);
+
+  const url = getPath();
+  const fetchProps = {
+    method: "GET",
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json",
+      "X-CSRFToken": "",
+    },
+  };
+
+  useEffect(() => {
+    fetch(url + "api/users/", fetchProps)
+      .then((res) => res.json())
+      .then(
+        (result) => {
+          if (result === []) {
+            setIsEmployee(false);
+          } else {
+            setIsEmployee(true);
+            setEmployees(result);
+          }
+
+          setIsLoaded(true);
+        },
+        (error) => {
+          console.error(error);
+        }
+      );
+  }, [props.count]);
+
```

Comment:
Consider to move AJAX request method into another component like the other requests,
so life-cycle could cover only changed part of component (as far as I know).
Plus in this situation [props.count] is not necessary ;)
Component loads once so ...
Most important think is - "it works" :-)
so I give +